trussworks/react-uswds
# test: Fix unit test and storybook console errors and warnings

# Summary

Originally started investigation due to https://github.com/trussworks/react-uswds/issues/1161, but that seems to have resolved itself with the most recent dependabot updates. Never the less, in the update to testing-library/user-event, there was a change to the way `userEvent.type()` works, which introduced a console error (but no failed tests) in our unit tests. Since I noticed it, I figured it would be good to just clean it up while there.

Additionally we had a couple of console warnings in Identifier tests and stories where the unique "key" prop was not being recognized by react. I updated those to no longer produce that warning as well.


## Related Issues or PRs

kind of related to https://github.com/trussworks/react-uswds/issues/1161 as noted above.

## How To Test

- `yarn test` and verify no error or warning logs appear
- In storybook, open the console and navigate to identifier components. Verify unique "key" warning is no longer present.
